gemfire-shared/src/main/java/com/gemstone/gemfire/internal/shared/SystemProperties.java

```diff
@@ -228,7 +228,8 @@ public static SystemProperties getClientInstance() {
 
   public static Callbacks getGFXDServerCallbacks() {
     try {
-      Class<?> factoryProvider = Class.forName(GFXD_FACTORY_PROVIDER);
+      Class<?> factoryProvider = Class.forName(GFXD_FACTORY_PROVIDER,
+          true, ClassLoader.getSystemClassLoader());
```

Comment:
Why was this required. Any other class loader getting used instead of system class loader? Just to understand.

Reply:
Yes this change has been reverted in next PR https://github.com/SnappyDataInc/snappy-store/pull/347/files#diff-834f00639cc7c1a04f4d4f8455251e42

Had added this because was testing a custom system loader that had a faster indexed class lookup from jars. However, dropped it because advantage was not significant (~1s for each node).